app/service/sender.py

```diff
@@ -7,7 +7,9 @@
 from app.notifications.process_notifications import persist_notification, send_notification_to_queue
 
 
-def send_notification_to_service_users(service_id, template_id, personalisation={}, include_user_fields=[]):
+def send_notification_to_service_users(service_id, template_id, personalisation=None, include_user_fields=None):
+    personalisation = personalisation or {}
+    include_user_fields = include_user_fields or []
```

Comment:
Did it actually catch this as PEP violation?

Reply:
nah, i've just got a hawklike vision for detail